tools-entry-scorer/src/main/java/org/uniprot/core/scorer/uniprotkb/ProteinDescriptionScored.java

```diff
@@ -154,7 +137,7 @@ private void addECsFromSubName(List<ProteinSubName> proteinNames, Set<EC> ecs) {
                         .collect(Collectors.toSet()));
     }
 
-    private List<Evidence> extractProteinAltNameEvidences(ProteinAltName name) {
+    private List<Evidence> extractProteinAltNameEvidences(ProteinName name) {
```

Comment:
I think that we might be able to merge this method with the Rec name equivalent.

Reply:
done